packages/dataviews/src/style.scss

```diff
@@ -542,11 +557,13 @@
 
 .dataviews-no-results,
 .dataviews-loading {
-	padding: 0 $grid-unit-40;
+	padding: 0 $grid-unit-60;
```

Comment:
I think it would be better to consider container queries for this padding as well. Otherwise, the left and right padding will not match when matching the container query.

![no-results-padding](https://github.com/WordPress/gutenberg/assets/54422211/e2202adf-9305-4cb6-9883-9df80b63ff3e)